fontforgeexe/charview.c

```diff
@@ -9812,14 +9812,14 @@ static void _CVMenuOverlap(CharView *cv,enum overlap_type ot) {
     if ( cv->b.sc->inspiro && hasspiro()) {
 	// Detection is not implemented for Spiro, so just clear them.
 	// TODO: Detect point survival in Spiro mode.
-	cv->lastselpt = NULL; cv->lastselcp = NULL;
+	cv->p.sp = cv->lastselpt = NULL; cv->p.spiro = cv->lastselcp = NULL;
```

Comment:
This is weird stylistically, seems like one chained assignment or three separate assignments would be preferable.

Reply:
I added a linebreak to resolve this in a way compatible with the parallel code below.